docs/paper/dev/getting-started/userdev.md

```diff
@@ -0,0 +1,87 @@
+---
+slug: /dev/userdev
+---
+
+# Paperweight Userdev
+
+**Paperweight** is the name of Paper's custom build tooling. The **Userdev** gradle plugin part of that
+provides access to non-api code (also called internals, or NMS) during development.
+
+:::note
+This guide is written using the Kotlin DSL for gradle and assumes you have some basic knowledge of gradle.
+If you want to see a fully-functioning plugin that uses **Userdev**,
+check out this [example plugin](https://github.com/PaperMC/paperweight-test-plugin).
+:::
+
+
+## Why this is useful
+The Paper server provided through the **paperclip** jars on the downloads page and API
+use Spigot's mappings, which are essentially some type names, but fully obfuscated fields and methods.
+This can make it hard to work with in a development environment. This plugin lets you use fully deobfuscated
+types, names and fields during development, and then remaps the plugin so it can still be used with the obfuscated
```

Comment:
```suggestion
This can make it hard to work with in a development environment. This Gradle plugin lets you use fully deobfuscated
types, names and fields during development, and then remaps your Paper plugin so it can still be used with the obfuscated
```

Just to clarify what plugin we are talking about - Userdev vs Paper

Reply:
Don't think I'm gonna use `Paper plugin` here cause that's a separate thing now. I think just adding `your` works

Reply:
That works too